Content.Server/GameObjects/Components/Suspicion/SuspicionRoleComponent.cs

```diff
@@ -188,8 +188,8 @@ void IExamine.Examine(FormattedMessage message, bool inDetailsRange)
             }
 
             var tooltip = IsTraitor()
-                ? Loc.GetString($"They were a [color=red]traitor[/color]!")
-                : Loc.GetString($"They were an [color=green]innocent[/color]!");
+                ? Loc.GetString("They were a [color=red]traitor[/color]!")
```

Comment:
Just do `Loc.GetString("They were a [color={0}]{1}[/color]!",color,Loc.GetString(role))` or smth like that